### PR TITLE
Text changes for new version of AR widgets

### PR DESCRIPTION
**Pull request by HedwigAR**
Soon we'll be replacing the current AR widgets with a version that has a new library behind it. This means most of the functionality will be the same with a few changes. 
Currently these are only the text changes, I still need to change the images.

### CODE REVIEW

File: content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets.md [hunk 1]
```diff
@@ -87,13 +80,7 @@ Visual widgets are widgets that add a virtual object to the scene.
...
 {{< figure src="/attachments/refguide/mobile/ar-widgets/3DObject.gif" alt="3DObject" >}}
 
-Beyond using simple 3D objects, there are also ways to add more complex and custom objects to your scene. The 3D object widget takes either a URL or a FileDocument (you have to upload its model later). There are currently 3 types supported: *.obj*, *.gltf*, and *.glb*. Each with their own benefits. Using a custom material like with the simple visual widgets is only supported in the *.obj* format.
-
-### 4.5 Text (AR)
-
-{{< figure src="/attachments/refguide/mobile/ar-widgets/Text.gif" alt="Text" >}}
-
-**Text (AR)** can be used to have 3D or 2D text floating in space. It is best practice to add only a few lines of text to the scene in this way. Bigger paragraphs are better communicated through a 2D text field on another place in your app.
+Beyond using simple 3D objects, there are also ways to add more complex and custom objects to your scene. The 3D object widget takes either a URL. There are currently 3 types supported: *.obj*, *.gltf*, and *.glb*. Each with their own benefits.
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets.md]: Is this sentence still meant to be "The 3D object widget takes either a URL or a FileDocument (you have to upload its model later)." ?

**Comment by Luyao-Zhang-1** [comment]
Currently, the Native Mobile AR widget has a very short [Marketplace guide](https://docs.mendix.com/appstore/modules/native-mobile-ar/) with only one introduction section. (Last year, we [tried](https://github.com/mendix/docs/pull/3524) to investigate if we can extend the doc, but because the technology would be switched out for the widgets soon, it didn't make much sense to update a doc that would become outdated soon. So we have been [waiting for an opportunity](https://mendix.atlassian.net/browse/TW-1405) to update the Marketplace guide with this new release.)

It seems that [AR Widget Overview](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets/) has most of the information that could be used in the Marketplace guide. @ConnorLand Would it make sense to move this doc to the Marketplace Guide space? The doc of course needs to be restructured to fit the MG template. We can add cross-references to this doc in [Get Started with AR](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-ar-simple-cube/) and [Create an AR Bigness Card](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-ar-business-card/). If you think it makes sense, I can convert it a Marketplace Guide if you like.

**Comment by ConnorLand** [comment]
> Currently, the Native Mobile AR widget has a very short [Marketplace guide](https://docs.mendix.com/appstore/modules/native-mobile-ar/) with only one introduction section. (Last year, we [tried](https://github.com/mendix/docs/pull/3524) to investigate if we can extend the doc, but because the technology would be switched out for the widgets soon, it didn't make much sense to update a doc that would become outdated soon. So we have been [waiting for an opportunity](https://mendix.atlassian.net/browse/TW-1405) to update the Marketplace guide with this new release.)
> 
> It seems that [AR Widget Overview](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets/) has most of the information that could be used in the Marketplace guide. @ConnorLand Would it make sense to move this doc to the Marketplace Guide space? The doc of course needs to be restructured to fit the MG template. We can add cross-references to this doc in [Get Started with AR](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-ar-simple-cube/) and [Create an AR Bigness Card](https://docs.mendix.com/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-ar-business-card/). If you think it makes sense, I can convert it a Marketplace Guide if you like.

I think that's a great idea. The AR Widget Overview doc is already written very much like a Marketplace guide. @HedwigAR do you foresee any problems with this? If not, I think Luyao should move the widget overview doc into Marketplace and enact the crosslinks she discussed.

**Comment by Luyao-Zhang-1** [comment]
@ConnorLand Thank you Connor! I just had a quick chat with Hedwig. She agrees with the change, so I will take over this doc and move it to the right place. Should I just work on this PR at the same time?

**Comment by ConnorLand** [comment]
> @ConnorLand Thank you Connor! I just had a quick chat with Hedwig. She agrees with the change, so I will take over this doc and move it to the right place. Should I just work on this PR at the same time?

@Luyao-Zhang-1 I'll finish doing basic proofs/edits and then hand this PR over for you. I'll Slack you when my proofs are done. You can do your document moves in here, and coordinate with Hedwig to achieve the rest of the PR accomplishes (new widget library updates).

**Comment by Luyao-Zhang-1** [comment]
> 
@ConnorLand  Sounds good! Thanks!

File: content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets.md (hunk 1 above)
Comment (HedwigAR) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/ar-widgets.md]: Ah, no, I made a mistake there, thanks! I'll fix it

**Pull request merged**